src/modlunky2/config.py

```diff
@@ -140,6 +140,7 @@ def clone(self: T) -> T:
 @dataclass
 class CategoryTrackerConfig(CommonTrackerConfig):
     always_show_modifiers: bool = skip_default_field(default=False)
+    excluded_categories: Optional[List[str]] = skip_default_field(default=None)
```

Comment:
This can be a `Set[MyEnum]`, provided `MyEnum` can be serialized. An `Enum` with `str` values should work well.

I think `FrozenSet` would also be fine. It's not necessary since we clone this object before sending it to the tracker's thread (where `RunState` and `RunLabel` do stuff). `Set` might be easier to work with, tho?